yarn-project/circuits.js/src/utils/call_wasm.ts

```diff
@@ -13,8 +13,8 @@ export async function callAsyncWasm<T>(
   // Allocate memory for the input buffer and the pointer to the pointer to the output buffer
   const inputBufPtr = wasm.call('bbmalloc', inputBuf.length);
   wasm.writeMemory(inputBufPtr, inputBuf);
-  if (method.indexOf('_rollup_') != -1) {
-    // TODO: Remove this once kernel circuits also return failures
+  if (!method.startsWith('private_kernel__')) {
+    // TODO: Remove this once private kernel circuits also return failures
```

Comment:
Should Davids styleguide be applied here with tagging the person handling the todo? Or issue etc

Reply:
I am removing this in the next PR!